cg/apps/crunchy.py

```diff
@@ -0,0 +1,320 @@
+    #     if self.spring_compression_done(
+    #         fastq_first_path=fastq_first_path, fastq_second_path=fastq_second_path
+    #     ):
+    #         return False
+    #
+    #     return True
+
+    @staticmethod
+    def get_flag_path(file_path):
+        return file_path.with_suffix(FLAG_PATH_SUFFIX)
+
+    @staticmethod
+    def get_index_path(file_path):
+        index_type = CRAM_INDEX_SUFFIX
+        if file_path.suffix == BAM_SUFFIX:
+            index_type = BAM_INDEX_SUFFIX
+        return file_path.with_suffix(file_path.suffix + index_type)
+
+    @staticmethod
+    def get_cram_path_from_bam(bam_path: Path) -> Path:
+        """ Get corresponding CRAM file path from bam file path """
+        if not bam_path.suffix == BAM_SUFFIX:
+            LOG.error("%s does not end with %s", bam_path, BAM_SUFFIX)
+            raise ValueError
+        cram_path = bam_path.with_suffix(CRAM_SUFFIX)
+        return cram_path
+
+    # @staticmethod
+    # def change_suffix_spring(fastq_first_path: Path, fastq_second_path: Path) -> Path:
+    #     """Make correct suffix for spring compressed fastq-files"""
+    #     if not fastq_first_path.name.endswith(FASTQ_FIRST_SUFFIX):
+    #         LOG.error("%s does not end with %s", fastq_first_path, FASTQ_FIRST_SUFFIX)
+    #         raise ValueError
+    #
+    #     if not fastq_second_path.name.endswith(FASTQ_SECOND_SUFFIX):
+    #         LOG.error("%s does not end with %s", fastq_second_path, FASTQ_SECOND_SUFFIX)
+    #         raise ValueError
+    #
+    #     if not str(fastq_first_path).replace(FASTQ_FIRST_SUFFIX, "") == str(
+    #         fastq_second_path
+    #     ).replace(FASTQ_SECOND_SUFFIX, ""):
+    #         LOG.error(
+    #             "%s and %s does not have the same root",
+    #             fastq_first_path,
+    #             fastq_second_path,
+    #         )
+    #         raise ValueError
+    #
+    #     spring_path = Path(
+    #         str(fastq_first_path).replace(FASTQ_FIRST_SUFFIX, SPRING_SUFFIX)
+    #     )
+    #     return spring_path
+
+    @staticmethod
+    def _get_slurm_header(
+        job_name: str,
+        log_dir: str,
+        account: str,
+        mail_user: str,
+        conda_env: str,
+        ntasks: int,
+        mem: int,
+    ) -> str:
+        sbatch_header = SBATCH_HEADER_TEMPLATE.format(
+            job_name=job_name,
+            account=account,
+            log_dir=log_dir,
+            conda_env=conda_env,
+            mail_user=mail_user,
+            ntasks=ntasks,
+            mem=mem,
+        )
+        return sbatch_header
+
+    @staticmethod
+    def _get_slurm_bam_to_cram(
+        bam_path: str, cram_path: str, flag_path: str, reference_path: str
+    ) -> str:
+        sbatch_body = SBATCH_BAM_TO_CRAM.format(
+            bam_path=bam_path,
+            cram_path=cram_path,
+            flag_path=flag_path,
+            reference_path=reference_path,
+        )
+        return sbatch_body
+
+    # @staticmethod
```

Comment:
branch?

Reply:
I can remove the commented code from this PR

Reply:
I think not done things should not go into production, I still thinks we should remove this dangling outcommented code, what do you say?